dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/QuayImageRegistry.java

```diff
@@ -354,6 +355,37 @@ public Registry getRegistry() {
         return Registry.QUAY_IO;
     }
 
+    @Override
+    public boolean canConvertToAuto(Tool tool) {
+        final String repo = tool.getNamespace() + '/' + tool.getName();
+        final Gson gson = new Gson();
+
+        // Grab build information for given repository
+        String urlBuilds = QUAY_URL + "repository/" + repo + "/build/?limit=" + Integer.MAX_VALUE;
```

Comment:
Hmmmm, let's create an issue for optimizing the performance of refresh with a profiler. 
Then comment with the issue number here, I think this will end up pulling the builds twice for updateAPIToolsWithBuildInformation and we're likely not using conditional requests (don't even know if quay.io supports it)